src/api/app/models/notification.rb

```diff
@@ -22,7 +22,7 @@ def user_active?
   end
 
   def any_user_in_group_active?
-    !subscriber.users.recently_seen.empty?
+    !subscriber.users.seen_since(3.months.ago).empty?
```

Comment:
This is the same as  `subscriber.users.seen_since(3.months.ago).any?` , no?

Reply:
True, addressing...